### ISSUE TITLE
`jenkins/jnlp-agent-jdk8` Docker image pushed 5 hours ago

### ISSUE THREAD

**Issue by lemeurherveCB** [jenkins-infra/helpdesk#4364]
### Service(s)

Docker Hub

### Summary

As discovered by bigon in https://matrix.to/#/!JLUOInpEYmxJIYXlzs:matrix.org/$yPevTYe0BwsopkDn_ny_8irLvuCzNiE9GUS752sq5ew?via=g4v.dev&via=gitter.im&via=matrix.org, a tag for this old image has been pushed 5 hours ago today: https://hub.docker.com/layers/jenkins/jnlp-agent-jdk8/latest/images/sha256-32430a167531170999188c7d7a294f9a7a685672cc6f16669b16bf1d2df07e7a?context=explore

Bigon also mentioned that this image is broken (who would have known? ^^)

```
docker run -ti --rm jenkins/jnlp-agent-jdk8:latest --help
Error: A JNI error has occurred, please check your installation and try again
Exception in thread "main" java.lang.UnsupportedClassVersionError: hudson/remoting/Launcher has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
```

**Comment by gounthar** [comment]
I suppose we can close, as https://github.com/jenkinsci/docker-inbound-agents/pull/47 fixed https://github.com/jenkinsci/docker-inbound-agents/issues/46.

**Comment by dduportal** [comment]
> I suppose we can close, as [jenkinsci/docker-inbound-agents#47](https://github.com/jenkinsci/docker-inbound-agents/pull/47) fixed [jenkinsci/docker-inbound-agents#46](https://github.com/jenkinsci/docker-inbound-agents/issues/46).

Thanks for fixing this issue @gounthar !
